src/components/Tasks/AddNewTaskItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useContext } from 'react';
+
+import ICAL from 'ical.js';
+import uuid from 'uuid';
+
+import Checkbox from '@material-ui/core/Checkbox';
+import TextField from '@material-ui/core/TextField';
+
+import { TaskType, TaskStatusType, TaskPriorityType } from '../../pim-types';
+import { ListItem } from '../../widgets/List';
+import { PimContext } from '../../Pim';
+
+const AddNewTaskItem = () => {
+  const [title, setTitle] = useState('');
+  const { onItemSave: save, collectionsTaskList: collections } = useContext(PimContext);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(e.target.value);
+  };
+
+  const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+
+      const event = new TaskType(null);
+      event.uid = uuid.v4();
+      event.title = title;
+      event.status = TaskStatusType.NeedsAction;
+      event.priority = TaskPriorityType.None;
+      event.lastModified = ICAL.Time.now();
+
+      save(event, collections[0].uid, undefined, false);
```

Comment:
It's missing error-handling. What if the save fails?

Reply:
This is true; however, there is not really error handling in any part of the app (at least in terms of saving items). Can I save this for a future PR and implement it app wide?

Reply:
I think item save has error handling, no? Anyhow, I see you took it out to its own PR, I'll comment there once I review it if I have anything more to add. Though if there's no error-handling anywhere else, I'm OK with not adding it here for now.